src/sentry/static/sentry/app/utils/integrationUtil.tsx

```diff
@@ -36,8 +36,7 @@ export const getAnalyticsSessionId = () =>
 export const getCategorySelectActive = (organization?: Organization) => {
   const variant = organization?.experiments?.IntegrationDirectoryCategoryExperiment;
   const localStore = localStorage.getItem(SHOW_INTEGRATION_DIRECTORY_CATEGORY_SELECT);
-
-  if (localStore !== undefined) {
+  if (localStore !== null) {
```

Comment:
nit: I think just `if (localStore)` would work for both right?

Reply:
I think the issue is if localstore is 0 and then we're checking the variant instead of localstorage.